app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -116,9 +115,8 @@ class SuggestedEditsTasksFragment : Fragment() {
                 }
                 .subscribe({ response ->
                     val editorTaskCounts = response.query()!!.editorTaskCounts()
-                    NotificationEditorTasksHandler.dispatchEditorTaskResults(requireContext(), editorTaskCounts!!)
                     var totalEdits = 0
-                    for (count in editorTaskCounts.descriptionEditsPerLanguage.values) {
+                    for (count in editorTaskCounts!!.descriptionEditsPerLanguage.values) {
```

Comment:
Can we move the `!!` to the end of the line `#117`?

`val editorTaskCounts = response.query()!!.editorTaskCounts()!!`